app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/MongoSendTask.java

```diff
@@ -94,10 +101,22 @@ public Void doInBackground(String... urls) {
                                         } else {
                                             Log.wtf(TAG, "Bloodtest with ID: " + up.reference_id + " appears to have been deleted");
                                         }
+                                    } else if (type.equals(Treatments.class.getSimpleName())) {
```

Comment:
This probably makes sense.. its a reminder that the treatment handling in the nighscout uploader was added independently before the uploader-queue framework existed and I haven't got round to moving it in to a proper place.